test-framework/test-suites/integration/tests/load/test_load.py

```diff
@@ -0,0 +1,45 @@
+import json
+from pathlib import Path
+import pytest
+
+# Running load appears to modify /etc/hosts.
+# We need to use this revert fixture for all tests to ensure we don't bork other tests running in parallel.
+# NOTE: This variable name must NOT be changed. See
+# https://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects for more info.
+pytestmark = pytest.mark.usefixtures("revert_etc")
```

Comment:
I'd like to see the function added to a class named class TestLoad and the pytest.mark.usefixtures used as a decorator, to keep things consistent:
```
@pytest.mark.usefixtures("revert_etc")
class TestLoad:
	@pytest.mark.parametrize(
		"dump_json, expected_results_json, list_command_template, object_name",
		(
			("load/json/partition.json", "load/json/expected_partition.json", "stack list {} storage partition output-format=json", "storage partition"),
		)
	)
	def test_load_processing(host, test_file, dump_json, expected_results_json, list_command_template, object_name):
	...
```

I did a poor job of documenting the style for creating tests here:
https://github.com/Teradata/stacki/tree/master/test-framework/test-suites/integration

Reply:
What's the value of the class in this case?

I can definitely see removing `pytestmark` at the module level and attaching it as an additional decorator to the test function though, now that I don't believe there will be multiple test functions in this file but rather a set of additional parametrizations added.

Reply:
The class is for consistency. And for when down the line someone does add another test function, such as checking for errors, they won't forget to create the class.

Also, unless the test logic is identical, don't go crazy with parametrize. Keep tests brain-dead simple. Tests (at least integration and unit) should never have advanced logic, if they do you probably need to split up the test to multiple smaller ones.

Reply:
Ok, I'll add the class. If you look at the linked draft PRs, they use the exact same test logic and just add additional input dump.json files and expected `stack list ...` output files.